.github/workflows/build-test-pr.yml

```diff
@@ -259,92 +260,20 @@ jobs:
           make pip_local
           make pytest_and_coverage
 
+      - name: Upload coverage report to Codecov
```

Comment:
Why do we upload these to Codecov?

Reply:
because the tool is cloud driven. Just following the suggested process for codecov.